src/ClassBindings.php

```diff
@@ -178,4 +190,14 @@ private function kebapToCamelCase(string $input)
 
         return $output;
     }
+
+    /**
+     * @return CallbackBinding[]
+     */
+    private function callbackBindings(): array
```

Comment:
maybe we can move the detection of callbackBindings to the `register` method and store them in a seperate array?

Reply:
👍 done